docker/Dockerfile

```diff
@@ -20,7 +21,8 @@ RUN gem install bundler
 COPY Gemfile /opt
 COPY Gemfile.lock /opt
 
-RUN bundle install
+RUN bundle config build.nokogiri --use-system-libraries \
+    && bundle install --without ${BUNDLE_WITHOUT}
 
 COPY . .
 RUN rake assets:precompile
```

Comment:
This step is unnecessary in development mode.

Reply:
It found the uglifier problem. So while it does add some additional image build time, I'd suggest that it is worth it to have dev as similar to prod as possible.

Reply:
But it only will find it when you docker build. (and you don't need to do docker build unless you change the Gemfile.lock). Our CI does docker-build and should find such problems.

Reply:
Still not clear on all this Rails, but doesn't this mean when Gemfile.lock changes and you need to do a rebuild, it will take an extra 30 seconds to run `rake assets:precompile`? Or am I misunderstanding?

Reply:
yes, it will take an extra 30 s to run `rake assets:precompile`, but since it's in development mode, it won't use the output of that. It's going to recompile the assets on every request.